crates/pallet-domains/src/lib.rs

```diff
@@ -161,6 +199,41 @@ mod pallet {
         OptionQuery,
     >;
 
+    #[pallet::storage]
+    pub(super) type NextOperatorId<T> = StorageValue<_, OperatorId, ValueQuery>;
+
+    #[pallet::storage]
+    pub(super) type OperatorIdOwner<T: Config> =
+        StorageMap<_, Identity, OperatorId, T::AccountId, OptionQuery>;
+
+    #[pallet::storage]
+    pub(super) type DomainStakingSummary<T: Config> =
+        StorageMap<_, Identity, DomainId, StakingSummary<OperatorId, BalanceOf<T>>, OptionQuery>;
+
+    #[pallet::storage]
+    pub(super) type OperatorPools<T: Config> =
+        StorageMap<_, Identity, OperatorId, OperatorPool<BalanceOf<T>, T::Share>, OptionQuery>;
+
+    #[pallet::storage]
+    pub(super) type Nominators<T: Config> = StorageDoubleMap<
```

Comment:
Seems this storage item never use.

Reply:
yes this is not used because we do not have epoch transition during which all pending items are added to this storage